wordpress-mobile/WordPress-iOS
# White Bars II: Serif fonts

Refs #14774.

This PR switches the navigation bars to use serif fonts. We use the system font on iOS 13+, and Noto on older versions. This needed some extra methods [added to WordPress-Shared-iOS here](https://github.com/wordpress-mobile/WordPress-iOS-Shared/pull/275).

![white-headers-ii](https://user-images.githubusercontent.com/4780/91716754-25c57a80-eb88-11ea-8bbb-73c88a078d9f.png)

**To test**

- Build and run
- Test on iOS 12 and 13.
- Do a smoke test of the app and verify that you see serif fonts for the navigation bar title throughout. In particular, also check the blog selector in the editor, and the header in reader detail views.

PR submission checklist:
